pkg/cmd/admin/groups/sync/rfc2307/ldapinterface.go

```diff
@@ -82,6 +87,11 @@ func (e *LDAPInterface) ExtractMembers(ldapGroupUID string) ([]*ldap.Entry, erro
 	// find members on LDAP server or in cache
 	for _, ldapMemberUID := range ldapMemberUIDs {
 		memberEntry, err := e.userEntryFor(ldapMemberUID)
+		if ldaputil.IsQueryOutOfBoundsError(err) {
```

Comment:
also, having just attempted a sync against a large employee group... it is super annoying when the entire sync fails because a group has an obsolete member entry in a directory outside your control.